en/docs/install-and-setup/setup/reference/supported-cipher-suites.md

```diff
@@ -2,40 +2,13 @@
 
 Given below are the cipher suites that are functional in Tomcat (Tomcat version 9 with the JSSE provider 11) for the TLSv1.2 protocols. 
```

Comment:
Do we need this statement? We might have to reword it since there isn't any cipher suites mentioned in the page anymore.

Reply:
Updated as follows:
<img width="746" alt="Screenshot 2023-01-05 at 5 40 12 PM" src="https://user-images.githubusercontent.com/5195851/210777334-230603f2-6882-4025-9231-536441b85ecc.png">